internal/work/dispatcher.go

```diff
@@ -319,3 +319,19 @@ func (d *Dispatcher) senderName(sender dbus.Sender) (string, error) {
 
 	return "", fmt.Errorf("cannot get name for sender: %v", sender)
 }
+
+func (d *Dispatcher) CancelMessage(command yggdrasil.Command) error {
+	// Send the message through the cancel interface
+	obj := d.conn.Object("com.redhat.Yggdrasil1.Worker1."+command.Content.Arguments["directive"],
+		dbus.ObjectPath(filepath.Join("/com/redhat/Yggdrasil1/Worker1/", command.Content.Arguments["directive"])))
+	call := obj.Call("com.redhat.Yggdrasil1.Worker1.Cancel", 0,
+		command.Content.Arguments["directive"],
+		command.MessageID,
+		command.Content.Arguments["messageID"])
+	if err := call.Store(); err != nil {
+		return fmt.Errorf("cannot call Cancel method on worker: %v", err)
```

Comment:
It would be useful to print here more information about canceling (e.g. worker name: `directive` and `messageID`)

Reply:
Do you mean on the error? Or adding a debug message before the `obj.Call`?

Reply:
I mean `fmt.Errorf()`.